crates/ibc/src/core/mod.rs

```diff
@@ -28,6 +28,10 @@ pub mod ics24_host;
 pub mod ics26_routing;
 
 pub mod context;
+pub mod handler;
 
 pub use context::ExecutionContext;
 pub use context::ValidationContext;
+
+pub use handler::execute;
+pub use handler::validate;
```

Comment:
Do we also want to reexport `context::ContextError` here?

Reply:
[a8eeff0](https://github.com/cosmos/ibc-rs/pull/257/commits/a8eeff04a19a06011028653181ebb602fc385373)